package/MDAnalysis/analysis/hbond_analysis.py

```diff
@@ -0,0 +1,199 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Did you intentionally _not_ put it in `analysis/hbonds`? 

We need to decide how to move forward with H-bonds, see #2238.

Reply:
For now, just to avoid any confusion (on my part) and to avoid having to think of a new name for the module. But if we do decide to go ahead with this new version then at some point it should probably be moved into analysis/hbonds.

Reply:
This is now `analysis.hydrogenbonds.hbond_analysis`